dogowner/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class Gender(models.TextChoices):
+    Male = 'M', 'Male'
+    Female = 'F', 'Female'
+    Unknown = 'UN', 'Unknown'
+
+
+class Dogowner(models.Model):
```

Comment:
The class name should be camelcase, see https://realpython.com/python-pep8/#naming-styles